Library/Formula/dnscrypt-proxy.rb

```diff
@@ -18,6 +18,7 @@ class DnscryptProxy < Formula
     depends_on "autoconf" => :build
     depends_on "automake" => :build
     depends_on "libtool" => :build
+    depends_on "minisign" => :build
```

Comment:
Why would this only be `head` (or `:build`)? Would the script not work for currently stable versions?